### DISCUSSION TITLE
attu ForbiddenError

### DISCUSSION THREAD

**Discussion by Uijeong97** [milvus-io/milvus#34492] [Q&A and General discussion]
I can't access milvus attu. ForbiddenError.
and I was using attu 2.3.2 and then upgraded to attu 2.4.2.
I haven't set anything like User, Role, etc.

```
 POST 200 /api/v1/milvus/connect 154.498 ms @ ...
GET /api/v1/users 403 ForbiddenError: Can not find your connection, please check your connection settings.
 GET 403 /api/v1/users 0.579 ms ...
 GET 304 /api/v1/milvus/version 0.368 ms ...
```

Do I need to set user?

But, when connecting with pymilvus client, it connects without any authentication such as user, token, password, etc.

**Comment by yhmo** [comment]
Wait @shanghaikid comment.

**Reply by shanghaikid** [reply]
@Uijeong97 please upgrade attu to 2.4.x for milvus 2.4

**Reply by Uijeong97** [reply]
@shanghaikid 

Umm.. I use attu 2.4.2

**Reply by shanghaikid** [reply]
what is your milvus version?

**Reply by Uijeong97** [reply]
milvus version is 2.4.5

**Reply by shanghaikid** [reply]
That doesn't make sense.  

GET /api/v1/users 403 ForbiddenError: Can not find your connection, please 

This call gets milvus user info.  

Does your python scripts get milvus user info as well?

**Reply by shanghaikid** [reply]
GET /api/v1/users 403 ForbiddenError: Can not find your connection, please

but I think there is a connection lost.  Does it keep showing this error?

**Reply by Uijeong97** [reply]
Yes. Every time I try to connect milvus, an authentication request is fired, and it seems to fail.

**Reply by shanghaikid** [reply]
Can you restart your attu container? I've never meet this error.

**Reply by imuhammadarsalan** [reply]
i am getting same error
